src/inmanta/model.py

```diff
@@ -203,8 +203,6 @@ def __init__(
     ) -> None:
         self.type = mytype
         lower = multi[0]
-        if lower is None:
-            lower = -1
```

Comment:
@wouterdb I assume you know what this is used for (`ModelExporter`). Could you double-check this will not break anything?

Reply:
ModelExporter isn't used by anything so far, so that can be broken,